connection.h

```diff
@@ -96,14 +97,19 @@ class UDPThread : public Thread {
 
     void setTimeoutTable(std::map<uint32_t,uint32_t> &timeoutTable);
     std::map<uint32_t,uint32_t>& getTimeoutTable();
+    bool setClientConnectionTimeoutSec(uint32_t sec);
 
   private:
     /* This function transmits m_frameBuffer */
     void transmitBuffer();
     void fireTimer();
     /* Returns the current timer value in us */
-    uint32_t getTimerValue();
-    void adjustTimer(uint32_t interval, uint32_t value);
+    uint32_t getTimerValue(int timerfd);
```

Comment:
i should actually but this should imply that a thread has a timer...is alwasys true? I don't think so.We could move them in a timer class with virtual method from we can get a linuxTimer..